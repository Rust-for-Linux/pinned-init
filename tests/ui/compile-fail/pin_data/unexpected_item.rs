use pin_init::*;

#[pin_data]
fn foo() {}

fn main() {}
