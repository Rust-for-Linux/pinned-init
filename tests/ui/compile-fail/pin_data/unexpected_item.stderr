error: expected `struct`
 --> tests/ui/compile-fail/pin_data/unexpected_item.rs:4:1
  |
4 | fn foo() {}
  | ^^
